docs/reference/symbolic.rst

```diff
@@ -0,0 +1,65 @@
+
+.. _symbolic_math:
+
+=============
+Symbolic Math
+=============
+
+You can construct a model, for example a CQM, from symbolic math, which is especially
+useful for learning and testing with small problems.
+
+dimod supports binary and integer variables:
+
+>>> from dimod import Binary, Integer
+>>> # Create binary variables
+>>> x = Binary('x')
+>>> y = Binary('y')
+>>> # Create integer variables
+>>> i = Integer('i')
+>>> j = Integer('j')
+
+Such variables are constructed as either BQMs or QMS, depending on the type of
+variable:
+
```

Comment:
I would break each of these steps out and explain exactly what's happening with text.

I would also include two more:

```
>> x
BinaryQuadraticModel({'x': 1.0}, {}, 0.0, 'BINARY')
```
emphasizes the "one-variable models, not variables" point.

```
>>> Spin('s') + Binary('x')
QuadraticModel({'s': 1.0, 'x': 1.0}, {}, -0.0, {'s': 'SPIN', 'x': 'BINARY'}, dtype='float64')
```
is important because the type of the object is being "promoted" to QM.

Reply:
Done